storage/statedb/hasher_test.go

```diff
@@ -0,0 +1,41 @@
+package statedb
+
+import (
+	"testing"
+
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/storage/database"
+	"github.com/stretchr/testify/assert"
+)
+
+func checkHasherHash(t *testing.T, idx int, tc *testNodeEncodingTC) {
+	hash := common.BytesToHash(tc.hash)
+
+	memDB := database.NewMemoryDBManager()
+	db := NewDatabase(memDB)
+	h := newHasher(nil)
+	defer returnHasherToPool(h)
+
+	hashed, cached := h.hash(tc.expanded, db, false)
```

Comment:
How about adding test cases for `hashRoot`?

Reply:
Added in [a87f5d8](https://github.com/klaytn/klaytn/pull/1843/commits/a87f5d8d4a1f4b0445674bb253855fcadf00f636).